core/static/styles/_content.scss

```diff
@@ -0,0 +1,143 @@
+%font--nowrap {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.font--nowrap {
+  @extend %font--nowrap;
+}
+
+
+// GENERAL CONTENT STYLES
+//------------------------------
+
+a,
+.link {
+  color: $active-blue;
+  cursor: pointer;
+  text-decoration: underline;
+
+  &:active,
+  &:hover {
+    color: $mid-blue;
+  }
+  
+  &:focus {
+    outline: $focus-yellow solid 3px;
+  }
+}
+
+h1 {
+  @include heading-l;
+}
+
+h2 {
+  @include heading-m;
+}
+
+h3 {
+  @include heading-s;
+}
+
+h4, h5, h6 {
+  // What should these be?
+  // For now, using next font size down, but bold.
+  // Expect this to change.
+  @include body-m;
+  font-weight: bold;
+}
+
+p {
+  font: inherit;
+  margin: 0 0 1em 0;
+}
+
+li {
+  font: inherit;
+  padding: 0 0 0 $spacing-3;
+  position: relative;
+}
+
+ul {
+  margin: 0 0 1em 0;
+  list-style: none;
+
+  li::before {
+    background-color: $stone;
+    border-radius: 4px;
+    content: "";
+    display: inline-block;
+    height: 8px;
+    left: 0;
+    position: absolute;
+    top: 0.65em;
+    width: 8px;
+  }
+}
+
+.button {
+  @include body-m;
+  @include button($active-blue, $active-blue-alt);
+  border-style: solid;
+  border-width: 2px;
+  box-shadow: none;
+  cursor: pointer;
+  display: inline-block;
+  font-weight: 800;
+  overflow: hidden;
+  padding: 10px $spacing-2;
+  text-decoration: none;
+
+  &[disabled="disabled"] {
+    @include button-disabled($active-blue-disabled, $white);
+  }
+  
+  &:focus {
+    outline: $focus-yellow solid 3px;
+  }
+}
+
+.button-cta {
+  @include button-ghost($active-blue, $white);
+}
+
+.navigation {
+  margin: 0;
+  li {
+    padding: 0;
+    &::before {
+      display: none;
+    }
+  }
+}
+
+.dead {
+  cursor: not-allowed;
+}
+
+
+// MOBILE
+@media (max-width: 420px) {
```

Comment:
ditto

Reply:
this is just a css thing idk why it does that. syntax is perfectly fine